homework_03/01_alexander_yordanov/tests.py

```diff
@@ -0,0 +1,53 @@
+import main_task
+import math 
+
+# simple test if we can create a point
+def create_point():
+    point = main_task.Point(1, 2)
+    assert isinstance(point, main_task.Point)
+
+# simple test if we can create a circle
+def create_circle():
+    center = main_task.Point(3 ,6)
+    r = 2
+    circle = main_task.Circle(center, r)
+    assert isinstance(circle, main_task.Circle)
+
+# test if one circle is on the same place like the other (in this case i use the same circle)
+def test_same_circles():
+    circle1 = main_task.Circle(main_task.Point(1.0, 2.0), 1.0)
+
```

Comment:
tecknically  you need more tests to catch all cases but it doesn't matter you are good